### PR TITLE
[no ticket] update getDisk

### PR DESCRIPTION
**Pull request by Qi77Qi**
Made `GoogleComputeService.getDisk` recover on 404s and return `F[Option[Disk]]`

**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ To depend on the `MockGoogle*` classes, additionally depend on:
 
 Contains utility functions for talking to Google APIs via com.google.cloud client library (more recent) via gRPC. 
 
-Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-google2" % "0.12-65bba14"`
+Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-google2" % "0.13-TRAVIS-REPLACE-ME"`
```
Comment (rtitle) [inline README.md:63]: I think https://github.com/broadinstitute/workbench-libs/pull/350 also bumped to 0.13, may need to resolve conflicts

File: google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleDiskInterpreter.scala [hunk 2]
```diff
@@ -40,11 +40,23 @@ private[google2] class GoogleDiskInterpreter[F[_]: StructuredLogger: Timer: Cont
 
   def getDisk(project: GoogleProject, zone: ZoneName, diskName: DiskName)(
     implicit ev: ApplicativeAsk[F, TraceId]
-  ): Stream[F, Disk] = {
+  ): F[Option[Disk]] = {
     val projectZoneDiskName = ProjectZoneDiskName.of(diskName.value, project.value, zone.value)
-    retryF(
```
Comment (rtitle) [inline google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleDiskInterpreter.scala:45]: Do we care about retrying for `getDisk`? Probably not I'm guessing

File: README.md (hunk 1 above)
Comment (gpcarr) [reply README.md:63]: I just merged that [PR](https://github.com/broadinstitute/workbench-libs/pull/350), it bumps to 0.13, sorry to create conflicts

**Review by rtitle — APPROVED** [review]
LGTM, does Leo code need to change when it adopts this?

File: google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleDiskInterpreter.scala (hunk 2 above)
Comment (Qi77Qi) [reply google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleDiskInterpreter.scala:45]: yea, I think google's sdk retry under the hood...I feel we should add retry when we really need to (so far I haven't run seen issues without explicit retry)

**Comment by Qi77Qi** [comment]
@rtitle Leo will probably have some minor change...should be easy to fix..I'm going to leave this version at 0.13 since @gpcarr just bumped the version

File: README.md (hunk 1 above)
Comment (Qi77Qi) [reply README.md:63]: yea..I'm thinking not bumping version in this PR since it's so close to the other one unless you guys feel strongly about this.

**Pull request merged**